json-schema/schema.json

```diff
@@ -298,6 +316,52 @@
         }
       }
     },
+    "variable": {
+      "title": "Variable Object",
+      "type": "object",
+      "required": [
+        "dimensions"
```

Comment:
Should there be more required? Just the dimensions seems not so useful without any additional information, right? In dimensions we require either extent or values...

Reply:
I don't know. I don't think that requiring `values` makes sense, as these will often be numerical values with no pre-defined set of values. Requiring extent if `values` isn't specified may make sense, as long as it's allowed to be null.

Reply:
Setting the extent completely to null values would basically remove the requirement again. Hmm... not sure.